hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -650,7 +650,7 @@ private OzoneManager(OzoneConfiguration conf, StartupOption startupOption)
 
       SecretKeyProtocol secretKeyProtocol =
           HddsServerUtil.getSecretKeyClientForOm(conf);
-      secretKeyClient = new DefaultSecretKeySignerClient(secretKeyProtocol);
+      secretKeyClient = new DefaultSecretKeySignerClient(secretKeyProtocol, "");
```

Comment:
Why "" is used instead of node info here?

Reply:
Yes, this will be done in HDDS-8721.

Reply:
Okay, then don't forget to override these ""s with real node info in next patches, One in OzoneManager, another in HddsDatanodeService. 
cc @ArafatKhan2198